res/controllers/Traktor-Kontrol-S3-hid-scripts.js

```diff
@@ -37,6 +37,11 @@ TraktorS3.PitchSliderRelativeMode = true;
 // button down.  Letting go will stop playback.
 TraktorS3.SamplerModePressAndHold = false;
 
+// By default, touching the jog wheel does nothing special.  When this option is true,
+// touching the job wheel enables scratch mode. Pressing the jog button still also
+// enables scratching.
+TraktorS3.WheelTouchScratching = false;
```

Comment:
You've still misunderstood. This should not be a static set-and-forget option. It should be a state toggled by the jog button. You can keep this preference option though for the user to set their on default state on startup.

Reply:
oh! got it